src/database.ts

```diff
@@ -203,4 +213,17 @@ export class Database<T extends Schema> {
 			}
 		}
 	}
+
+	// given a bitmask, we decode and return the types
+	decodeType(bitmask: bigint): Set<number> {
```

Comment:
not a static function as we would want an instance of `database`,  even though this won't vary between various indexfiles.